### DISCUSSION TITLE
Fix: Vuetify's VSelect bug not showing items

### DISCUSSION THREAD

**Discussion by andy9t7** [Kitware/trame#397] [General]
Hi Trame Community,

I came across a bug where VSelect will not show items and shows instead [object Object]. To explain, in some of the trame examples, you assign items inside vuetify's VSelect widget by dictionaries with "text" key and then the corresponding "value" key. This will cause problems in your VSelect widget. 

**To fix, change "text" to "title".**

```
vuetify3.VSelect(
            # Representation
            v_model=("mesh_representation", Representation.Surface),
            items=(
                "representations",
                [
                    {"title": "Points", "value": 0},
                    {"title": "Wireframe", "value": 1},
                    {"title": "Surface", "value": 2},
                    {"title": "SurfaceWithEdges", "value": 3},
                ],
            ),
            label="Representation",
            hide_details=True,
            dense=True,
            outlined=True,
        )
```

I will post this here in case anyone has the same issue.

Kind regards,

Andy

**Comment by jourdain** [comment]
Not really a bug, just a different API between [v2](https://v2.vuetifyjs.com/en/api/v-select/#props) and [v3](https://vuetifyjs.com/en/api/v-select/#props-items).

If you wanted to keep the same structure, you could have set `item_title="text"`

**Comment by jourdain** [comment]
The migration guide capture [that here](https://vuetifyjs.com/en/getting-started/upgrade-guide/#v-select-v-combobox-v-autocomplete)
